octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po

```diff
@@ -772,50 +772,73 @@ msgstr ""
 msgid "Starts the laser job"
 msgstr "Startet den Laserjob"
 
-#: octoprint_mrbeam/static/js/mother_viewmodel.js:384
+#: octoprint_mrbeam/static/js/mother_viewmodel.js:61
+msgid "The laserjob will be cancelled."
+msgstr "Der Laservorgang wird abgebrochen."
+
+#: octoprint_mrbeam/static/js/mother_viewmodel.js:62
+msgid "Do you still want to cancel?"
+msgstr "Willst du immer noch abbrechen?"
+
+#: octoprint_mrbeam/static/js/mother_viewmodel.js:63
+#: octoprint_mrbeam/templates/settings/maintenance_settings.jinja2:136
+msgid "No"
+msgstr "Nein"
+
```

Comment:
When I tested this branch, Yes and no translations were already there. Do we need to add them again?

Reply:
There was a new place found and added to this translation, you can see that it was removed on line 5403